addons/main/functions/debug/fnc_debugDraw.sqf

```diff
@@ -168,12 +138,13 @@ private _fnc_DrawRect = {
             drawIcon3D ["a3\ui_f\data\Map\Markers\System\dummy_ca.paa", [1, 1, 1, 1], ASLtoAGL(_lastSeen), 1, 1, 0, "Last Seen Position"];
 
             drawLine3D [_headPos, _currentTarget call CBA_fnc_getPos, [1, 0, 0, 1]];
-            ["None", name _currentTarget] select (isNull _currentTarget);
+            [name _currentTarget, "None"] select (isNull _currentTarget);
         } else {
             _targetKnowledge append [
-               "Target Knowledge:<br/>",
-               "    Last Seen: N/A (N/A)<br/>",
-               "    Position Error: N/A<br/>"
+                "","",""
```

Comment:
just remove this append in total. this does not make any difference. L143-L148 can be removed